cmd/oc/oc.go

```diff
@@ -100,3 +104,18 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func installNonCRDSecurity(scheme *apimachineryruntime.Scheme) error {
```

Comment:
Although this is somewhat valid approach, I think we should rather split the installation at the API level, iow. inside openshift/api's security group we should have two separate calls for installing all resources (including CRDs) and just native ones. IIRC security is the only group that holds both and having the distinction at API level will prevent diverge in installing those when new resources will be added. /cc @deads2k @sttts

Reply:
The right solution would be to make kubectl/oc stop using the scheme membership as an indication of the patch type.

Reply:
Yeah, that'd be nice too.